### PR TITLE
Modified the global default viewport size for the app back to the original size

### LINKED ISSUE #10513: Global: Reset the default viewport size for the App
# Overview
The new viewport default size is causing some issues with how our pages are rendering. Until we're able to fix all of these responsiveness issues, let's revert the default viewport size to: 1020x768px

# Acceptance Criteria

- [ ] The default viewport for the app is 1020x768px

### PR DESCRIPTION
**Pull request by aptrons22**
# Overview:
I have added V8 configuration details for the default app size to be the original size.
Closes #10513

# Changelog:
Edited default app window width and height in V8 config migration

# Review requests:
In the app you will see the original default app size.

# Risk assessment:
Low

### CODE REVIEW

File: app-shell/src/config/__fixtures__/index.ts [hunk 1]
```diff
@@ -130,3 +131,14 @@ export const MOCK_CONFIG_V7: ConfigV7 = {
...
   },
 }
+
+export const MOCK_CONFIG_V8: ConfigV8 = {
+  ...MOCK_CONFIG_V7,
+  version: 8,
+  ui: {
+    ...MOCK_CONFIG_V7.ui,
+    width: 1024,
+    minWidth: 600,
```
Comment (b-cooper) [inline app-shell/src/config/__fixtures__/index.ts]: this line is unnecessary as `minWidth` from v7 is already set to 600

File: app-shell/src/config/migrate.ts [hunk 2]
```diff
@@ -187,14 +188,30 @@ const toVersion7 = (prevConfig: ConfigV6): ConfigV7 => {
...
 
+// config version 8 migration and defaults
+const toVersion8 = (prevConfig: ConfigV7): ConfigV8 => {
+  const nextConfig = {
+    ...prevConfig,
+    version: 8 as const,
+    ui: {
+      ...prevConfig.ui,
+      width: 1024,
+      minWidth: 600,
```
Comment (b-cooper) [inline app-shell/src/config/migrate.ts]: same comment here about not including the `minWidth` in this migration as it's left unchanged

File: app/src/redux/config/schema-types.ts [hunk 3]
```diff
@@ -136,4 +136,11 @@ export interface ConfigV7 extends Omit<ConfigV6, 'version'> {
   }
 }
 
-export type Config = ConfigV7
+export interface ConfigV8 extends Omit<ConfigV7, 'version'> {
+  version: 8
+  ui: ConfigV7['ui'] & {
+    minWidth: number
+  }
```
Comment (b-cooper) [inline app/src/redux/config/schema-types.ts]: the `ui` key is unchanged between v7 and v8, so these three lines should be unnecessary

File: app-shell/src/config/__fixtures__/index.ts (hunk 1 above)
Comment (aptrons22) [reply app-shell/src/config/__fixtures__/index.ts]: Thanks for the feedback I have made all of the corrections.  Now ready for review.

File: app-shell/src/config/__tests__/migrate.test.ts [hunk 4]
```diff
@@ -16,63 +17,71 @@ describe('config migration', () => {
...
     const v6Config = MOCK_CONFIG_V6
     const result = migrate(v6Config)
 
-    expect(result.version).toBe(7)
-    expect(result).toEqual(MOCK_CONFIG_V7)
+    expect(result.version).toBe(8)
+    expect(result).toEqual(MOCK_CONFIG_V8)
   })
 
   it('should keep version 7', () => {
```
Comment (vabruzzo) [inline app-shell/src/config/__tests__/migrate.test.ts]: the description of this test should change now to indicate version 7 should migrate to v8, similar to the tests above

Comment (aptrons22) [reply app-shell/src/config/__tests__/migrate.test.ts]: Yes that has been done.  Thanks

Comment (vabruzzo) [reply app-shell/src/config/__tests__/migrate.test.ts]: I'm not seeing that change here in this PR, have you pushed recent changes? In particular I'm referring to line 72. It reads here `should keep version 7`, but that's not what is being tested after the edits on lines 76-83. it should now read  `should migrate version 7 to latest`

Comment (aptrons22) [reply app-shell/src/config/__tests__/migrate.test.ts]: Yes I was working on a different branch.  I just made the update now and the CI test are running.  Thanks

**Review by vabruzzo — APPROVED** [review]
Looks good to me.

**Review by b-cooper — APPROVED** [review]
📏 📐

**Pull request merged**